omod-common/src/main/java/org/openmrs/module/webservices/docs/SearchHandlerDoc.java

```diff
@@ -94,13 +95,8 @@ public void setSearchQueriesDoc(List<SearchQueryDoc> searchQueriesDoc) {
 	}
 	
 	private String extractResourceName(String supportedResourceUrl) {
-		
```

Comment:
Yes, It is not used within the module. I could either remove the method or close the PR as it is not that important. What do you think?